### DISCUSSION TITLE
Data is missing after compression

### DISCUSSION THREAD

**Discussion by Nisargparekh7** [Dr-Emann/applesauce#42] [General]
issue I've seen is that when I compress a full folder (70gb folder size- made of various file sizes ranging from 1gb-4gb), few files were completely empty and data was missing after compression.

**Comment by Dr-Emann** [comment]
I believe #102 should have fixed any possible missing data. I'd also recommend trying with `--verify` (which will re-read the compressed file and ensures it has identical contents as the original file before replacing the original file with the compressed file) if you want to be even more sure.
